code/modules/projectiles/projectile/beams.dm

```diff
@@ -27,17 +27,24 @@
 	wound_bonus = -30
 	bare_wound_bonus = 40
 
-//overclocked laser, does a bit more damage but has much higher wound power (-0 vs -20)
+//Hellfire laser, ignites, but does less wounding
 /obj/projectile/beam/laser/hellfire
 	name = "hellfire laser"
-	wound_bonus = 0
-	damage = 25
-	speed = 0.6 // higher power = faster, that's how light works right
+	wound_bonus = -5 //Yeah, fire burns flesh
+	damage = 20
+	speed = 0.6 // Fire goes brrrr
```

Comment:
this comment is not useful and should not exist

Reply:
Done